fig/tendermint/liquidity/v1/liquidity.proto

```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
 
-package liquidity.v1;
+package fig.tendermint.liquidity.v1;
```

Comment:
This file was also copied from somewhere, I think it shouldn't be scoped under `fig/tendermint`.